CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ For information on changes in released versions of Teku, see the [releases page]
 ### Breaking Changes
 - Removed references to beacon block methods `/eth2/beacon_chain/req/beacon_blocks_by_root` and `/eth2/beacon_chain/req/beacon_blocks_by_range`.
 - `--Xmetrics-block-timing-tracking-enabled` option has been removed. The `--metrics-block-timing-tracking-enabled` option should be used instead (enabled by default).
+- `--Xbeacon-liveness-tracking-enabled` option has been removed. The `--beacon-liveness-tracking-enabled` option should be used instead (disabled by default).
```

Comment:
i wonder if it would make sense for this to be the same flag, so you enable doppelganger protection on BN and on VC, it might make the feature easier to understand and use? Just a thought, open to ideas on that...

Reply:
this is really the same as the other PR?